prod/westeurope/notifications/functions_pushnotif/function_app/terragrunt.hcl

```diff
@@ -85,6 +85,9 @@ inputs = {
 
     SLOT_TASK_HUBNAME = "ProductionTaskHub"
 
+    // activity default retry attempts
+    RETRY_ATTEMPT_NUMBER = 10
```

Comment:
```suggestion
    RETRY_ATTEMPT_NUMBER = 10
    
    APPINSIGHTS_INSTRUMENTATIONKEY = application_insights_instrumentation_key
    APPINSIGHTS_SAMPLING_PERCENTAGE = 5
```

Reply:
`APPINSIGHTS_SAMPLING_PERCENTAGE` added in https://github.com/pagopa/io-infrastructure-live-new/pull/492/commits/02fbd251e76cea7e2851dfdf76d5c99887e96e79

Reply:
`APPINSIGHTS_INSTRUMENTATIONKEY = application_insights_instrumentation_key`

this is already setted